### PR TITLE
4d

### PR DESCRIPTION
**Pull request by daron1337**
Added in main stack object:
- numberOfTemporalPositions (number of frames for each image)
- is4D (boolean field)

If the dicom is a timeserie, the stack will be ordered with image position and using the contentTime dicom tag. The resulting array of ordered images will be ImageSlice(0)-Time(0), ..., ImageSlice(0)-Time(N), ... ImageSlice(N)-Time(N). Scrolling a stack will result in visualize an image from time 0 to N and then the following image (by image position).

Added in larvitar store (viewport):
- minTimeId, timeId and maxTimeId: id from 0 to N according to numberOfTemporalPositions. TimeId is the current id visualized.
- timestamp which represent the current timestamp visualized.
- Update this fields when stackscrollwheel event is triggered or when the updateImage method is called
- timestamps and timeIds arrays are used to find the corresponding timeId or timestamp with imageIndex

- Fixed nnrd parsing for RGB Images

### CODE REVIEW

File: docs/examples/nrrd.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
       </div>
     </div>
 
-    <script src="./larvitar.js"></script>
+    <script src="../../dist/larvitar.js"></script>
```
Comment (ronzim) [inline docs/examples/nrrd.html:65]: restore original

**Comment by ronzim** [comment]
In this way, both mouse scrolling and slider won't work properly, right ?

**Comment by ronzim** [comment]
> Scrolling a stack will result in visualize an image from time 0 to N and then the following image (by image position).
This is not as expected 😁

**Comment by daron1337** [comment]
> > Scrolling a stack will result in visualize an image from time 0 to N and then the following image (by image position).
> 
> This is not as expected 😁

In all dicom viewers (eg osirix) 4D images scrolling works in this way :)

**Review by ronzim — APPROVED** [review]

**Pull request merged**